polymath/polynomial.py

```
@@ -487,26 +591,65 @@ def eval(self, x, recursive=True):
+            Scalar: A Scalar of values. The shapes of self and x are broadcasted
+                together following NumPy broadcasting rules. If self has shape (m, n) and
+                x has shape (p,), the result will have the broadcasted shape.
         """
 
         if self.order == 0:
+            # For zero-order polynomial, extract the constant value
+            # self._values has shape (..., 1) for the constant coefficient
+            # Extract the scalar value by indexing the last axis
+            tail_indx = self._drank * (slice(None),)
+            if tail_indx:
+                const_values = self._values[(Ellipsis, 0) + tail_indx]
```

Ruff (0.14.7)

Consider `(Ellipsis, 0, *tail_indx)` instead of concatenation

Replace with `(Ellipsis, 0, *tail_indx)`

(RUF005)